static/app/components/events/interfaces/crashContent/stackTrace/rawContent.spec.tsx

```diff
@@ -122,39 +122,76 @@ describe('RawStacktraceContent', function () {
       );
     });
 
-    const data_with_non_in_app: StacktraceType = {
```

Comment:
snakecase begone!

Reply:
Ha - not mine, but I hadn't even noticed! All this time programming in python has corrupted me!